lib/chef/resource/windows_security_policy.rb

```diff
@@ -80,13 +80,23 @@ class WindowsSecurityPolicy < Chef::Resource
       property :secvalue, String, required: true,
       description: "Policy value to be set for policy name."
 
+      load_current_value do |desired|
+        security_option_values = load_security_options_state
+        output = powershell_out(security_option_values).stdout.strip
```

Comment:
why powershell_out here instead of powershell_exec which would do all the JSON conversion for you?